docs/user-guide/concepts/data-types/categoricals.md

```diff
@@ -0,0 +1,260 @@
+# Categorical Data
+
+Categorical data represents string data where the values in the column have a finite set of values (usually way smaller than the length of the column). You can think about columns on gender, countries, currency pairings, etc. Storing these values as plain strings is a waste of memory and performance as we will be repeating the same string over and over again. Additionally, in the case of joins we are stuck with expensive string comparisons.
+
+That is why Polars supports encoding string values in dictionary format. Working with categorical data in Polars can be done with two different DataTypes: `Enum`,`Categorical`. Both have their own use cases which we will explain further on this page.
+First we will look at what a categorical is in Polars.
+
+In Polars a categorical is a defined as a string column which is encoded by a dictionary. A string column would be split into two elements: encodings and the actual string values.
+
+<table>
+<tr><th>String Column </th><th>Categorical Column</th></tr>
+<tr><td>
+<table>
+    <thead>
+        <tr>
+            <th>Series</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Polar Bear</td>
+        </tr>
+        <tr>
+            <td>Panda Bear</td>
+        </tr>
+        <tr>
+            <td>Brown Bear</td>
+        </tr>
+        <tr>
+            <td>Panda Bear</td>
+        </tr>
+        <tr>
+            <td>Brown Bear</td>
+        </tr>
+        <tr>
+            <td>Brown Bear</td>
+        </tr>
+        <tr>
+            <td>Polar Bear</td>
+        </tr>
+    </tbody>
+</table>
+</td>
+<td>
+<table>
+<tr>
+<td>
+
+<table>
+    <thead>
+        <tr>
+            <th>Physical</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>0</td>
+        </tr>
+        <tr>
+            <td>1</td>
+        </tr>
+        <tr>
+            <td>2</td>
+        </tr>
+        <tr>
+            <td>1</td>
+        </tr>
+        <tr>
+            <td>2</td>
+        </tr>
+        <tr>
+            <td>2</td>
+        </tr>
+        <tr>
+            <td>0</td>
+        </tr>
+    </tbody>
+</table>
+
+</td>
+<td>
+<table>
+    <thead>
+        <tr>
+            <th>Categories</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Polar Bear</td>
+        </tr>
+        <tr>
+            <td>Panda Bear</td>
+        </tr>
+        <tr>
+            <td>Brown Bear</td>
+        </tr>
+    </tbody>
+</table>
+</td>
+</tr>
+</table>
+</td>
+</tr>
+</table>
```

Comment:
Shall we use a Markdown table here? This syntax isn't very readable.

Same for the table below.

Reply:
Markdown does AFAIK not support tables within tables